openlibrary/plugins/openlibrary/js/add-book.js

```diff
@@ -1,5 +1,9 @@
 import {parseIsbn, isFormatValidIsbn10, isChecksumValidIsbn10, isFormatValidIsbn13, isChecksumValidIsbn13} from './edit.js'
 
+let invalidChecksum;
+let invalidIsbn10;
+let invalidIsbn13;
```

Comment:
This would be better as a class ; state variables like these can add extra complexity (namely making testing more difficult and clouding who the "owner" of state data is). But a lot of our JS is like this, so not a blocker.